src/components/map/layers/edit/RoutePathNeighborLinkLayer.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import { Polyline } from 'react-leaflet';
+import { inject, observer } from 'mobx-react';
+import IRoutePathLink from '~/models/IRoutePathLink';
+import INode from '~/models/INode';
+import { RoutePathStore, NeighborToAddType } from '~/stores/routePathStore';
+import { MapStore, NodeLabel } from '~/stores/mapStore';
+import RoutePathNeighborLinkService from '~/services/routePathNeighborLinkService';
+import INeighborLink from '~/models/INeighborLink';
+import NodeMarker from '../mapIcons/NodeMarker';
+import * as s from './routePathNeighborLinkLayer.scss';
+
+const USED_NEIGHBOR_COLOR = '#0dce0a';
+const UNUSED_NEIGHBOR_COLOR = '#fc383a';
+
+interface IRoutePathLayerProps {
+    routePathStore?: RoutePathStore;
+    mapStore?: MapStore;
+}
+
+@inject('routePathStore', 'mapStore')
+@observer
+class RoutePathNeighborLinkLayer extends Component<IRoutePathLayerProps> {
+    public hasNodeOddAmountOfNeighbors = (node: INode) => {
+        const routePath = this.props.routePathStore!.routePath;
+        return routePath!.routePathLinks!.filter(x => x.startNode.id === node.id).length
+            !== routePath!.routePathLinks!.filter(x => x.endNode.id === node.id).length;
+    }
+
+    private renderNeighborNode = (node: INode, neighborLink: INeighborLink, key: number) => {
+        return (
+            <NodeMarker
+                key={`${key}-${node.id}`}
+                isSelected={this.props.mapStore!.selectedNodeId === node.id}
+                onClick={this.addNeighborLinkToRoutePath(neighborLink.routePathLink)}
+                markerClasses={[s.neighborMarker]}
+                forcedVisibleNodeLabels={[NodeLabel.longNodeId]}
+                color={neighborLink.usages.length > 0 ? USED_NEIGHBOR_COLOR : UNUSED_NEIGHBOR_COLOR}
+                node={node}
+            >
+                <div className={s.usageAmount}>
```

Comment:
usageCount

http://crosstalk.cell.com/blog/the-difference-between-number-and-amount

Reply:
"Although amount and number both refer to quantity, and although plenty of people use them interchangeably, there is a difference.
Number refers to how many of something there is: how many mice, how many mutations, or how many calcium channels, for example. In other words, number refers to items we can count.

By contrast, amount refers to how much of something there is: how much stimulation, how much resistance, or how much liquid. In effect, amount refers to quantities we can measure but not individually count."